src/raftstore/store/snap.rs

```diff
@@ -700,7 +700,15 @@ impl Snap {
         for cf in SNAPSHOT_CFS {
             self.switch_to_cf_file(cf)?;
             let (cf_key_count, cf_size) = if plain_file_used(cf) {
-                let file = self.cf_files[self.cf_index].file.as_mut().unwrap();
+                // If the relative files are deleted after `Snap::new` and
+                // `init_for_building`, the file could be None.
+                let file = match self.cf_files[self.cf_index].file.as_mut() {
+                    Some(f) => f,
+                    None => {
+                        let e = box_err!("cf_file is none for cf {}", cf);
```

Comment:
Any test cover the failed situation?

Reply:
No. But I prefer merge it first because it's an urgent bug fix if it doesn't bring other side effects.

Reply:
you can add failpoint to test it. 

Does any user meet this problem?

Reply:
The problem can only happen when leader send one snapshot to 2 replicas concurrently. So consider these 2 cases:

1. Add 2 node/learner concurrently. It's prohibited by pd scheduler.
2. Add a node/learner on a region with 2 active, 1 pending replicas. However, add node will be reject by TiKV because of safety rules.

So I think the problem is triggered when we add a learner on a region with 2 active, 1 pending peers. So we have never meet it previously.

Reply:
Should we also need to consider sst format files?